src/libraries/System.Drawing.Common/src/Resources/Strings.resx

```diff
@@ -353,6 +353,9 @@
   <data name="PlatformNotSupported_Drawing" xml:space="preserve">
     <value>System.Drawing is not supported on this platform.</value>
   </data>
+  <data name="PlatformNotSupported_Unix" xml:space="preserve">
+    <value>System.Drawing is not supported on non-Windows platforms. See https://aka.ms/systemdrawingnonwindows for more information.</value>
```

Comment:
System.Drawing.Primitives is supported.  Can we be more specific in the error message?

Reply:
Sure. I'll add `.Common` to it. 
```suggestion
    <value>System.Drawing.Common is not supported on non-Windows platforms. See https://aka.ms/systemdrawingnonwindows for more information.</value>
```